src/components/content-presentation/do-and-dont-list/DoAndDontList.tsx

```diff
@@ -1,8 +1,8 @@
 'use client';
 import React, { HTMLProps, createContext, useContext, ReactNode } from 'react';
 import classNames from 'classnames';
-import { Tick, Cross } from '../../icons';
-import HeadingLevel, { HeadingLevelType } from '../../../util/HeadingLevel';
+import { Tick, Cross } from '@components/icons';
+import HeadingLevel, { HeadingLevelType } from '@util/HeadingLevel';
```

Comment:
Could you update names in this - I missed them as part of updating this component - they should be DoAndDontList

Reply:
Done